src/sentry/db/models/manager/base.py

```diff
@@ -257,11 +217,11 @@ def __value_for_field(self, instance, key):
         field = instance._meta.get_field(key)
         return getattr(instance, field.attname)
 
-    def contribute_to_class(self, model, name):
+    def contribute_to_class(self, model: Any, name: str) -> None:
         super().contribute_to_class(model, name)
         class_prepared.connect(self.__class_prepared, sender=model)
 
-    def get_from_cache(self, **kwargs) -> Model:  # TODO(typing): Properly type this
+    def get_from_cache(self, **kwargs: Any) -> Model:  # TODO(typing): Properly type this
```

Comment:
do you consider this properly typed now, or is the comment for later?

Reply:
Yeah the comment is for later. I think the comment might be referring to the `Model` return type.